### PR TITLE
Improve format and JUnit test of list command 

### PR DESCRIPTION
**Pull request by wjunjie01**
Improve format and JUnit test of list command 
Add assert and log statement in list command

### CODE REVIEW

**Comment by cheeseong2001** [comment]
LGTM

File: src/main/java/seedu/stockpal/commands/ListCommand.java [hunk 1]
```diff
@@ -3,9 +3,17 @@
 import seedu.stockpal.data.ProductList;
 import seedu.stockpal.ui.Ui;
 
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import static seedu.stockpal.common.Messages.MESSAGE_EMPTY_LIST;
+import static seedu.stockpal.common.Messages.MESSAGE_LIST_SUCCESS;
```
Comment (leongxingyu) [inline src/main/java/seedu/stockpal/commands/ListCommand.java]: Would it be better to import common.Messages instead of importing the messages to print?

**Review by Kobot7 — COMMENTED** [review]
LGTM

**Comment by leongxingyu** [comment]
LGTM

**Pull request merged**